pkg/kubectl/cmd/completion.go

```diff
@@ -103,6 +104,7 @@ func NewCmdCompletion(f cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 		ValidArgs: shells,
 	}
+	cmd.Flags().String("boiler-plate", "", "The name of the copyright header file, defaults to hardcoded header value.")
```

Comment:
a bit odd to have this parameter here. Would have expected some way to change the boilerplate per command such that downstream like oc can customize that.